tests/storages_tests/rdb_tests/alembic_tests/create-db.py

```diff
@@ -0,0 +1,43 @@
+import sys
+
+import optuna
+
+
+def objective(trial):
+    x = trial.suggest_uniform("x", -5, 5)  # optuna==0.9.0 does not have suggest_float.
```

Comment:
Just curious but were you able to generate the database files for all versions using the same single script?

Reply:
Yes, we can generate the database files from 0.9.0 to the master with this script.